src/index.html

```diff
@@ -17,6 +17,79 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="index.html"
+              class="link link-1"
```

Comment:
you don't need different class for all elements (link-1, link-2... itp) 
link is enough